js-http/fairos-http-client.js

```diff
@@ -0,0 +1,623 @@
+// Let us define properties needed for API calls
+var username = "user_"+(Date.now() / 1000).toFixed(0)
+var password = "159263487"
+var podName = "pod1"
+var host = "https://localhost:9090/v1/";
```

Comment:
for me `https` does not work on localhost
```
fairos-http-client.js:84 GET https://localhost:9090/v1//user/present?user_name=user_1643078197 net::ERR_SSL_PROTOCOL_ERROR
```

Reply:
The trailing slash seems to be causing some issues aswel

Reply:
Yes, probably that caused the cookie issue as well. I wasn't able to test it on localhost.